### PR TITLE
Feat enable return clause after set clause

### PR DESCRIPTION
**Pull request by htlim**
* Add eagerness operation for SET/REMOVE/DELETE

### CODE REVIEW

File: src/include/nodes/plannodes.h [hunk 1]
```diff
@@ -698,6 +698,8 @@ typedef struct Sort
 typedef struct Eager
 {
 	Plan		plan;
+	List	   *modifylist;
```
Comment (gtyun) [inline src/include/nodes/plannodes.h]: The others are 'modified list' but this is the only one 'modify list'.

File: src/test/regress/expected/cypher_dml.out [hunk 2]
```diff
@@ -1855,20 +1855,25 @@ MATCH p=(a)-[:supported]->() RETURN properties(a) AS a ORDER BY a;
 --
 MATCH (a) DELETE a;
 ERROR:  vertex 1 in "repo" has edge(s)
-MATCH p=()-[:lib]->() DETACH DELETE (vertices(p))[2];
+--MATCH p=()-[:lib]->() DETACH DELETE (vertices(p))[2];
```
Comment (gtyun) [inline src/test/regress/expected/cypher_dml.out]: Please do not comment test queries. If it is necessary, write description together.

File: src/backend/executor/nodeModifyGraph.c [hunk 3]
```diff
@@ -1137,7 +1138,9 @@ ExecSetGraph(ModifyGraphState *mgstate, GSPKind kind, TupleTableSlot *slot)
 
 		MemoryContextSwitchTo(oldmctx);
 
-		setSlotValueByName(result, newelem, gsp->variable);
+		attrno = findAttrInSlotByName(result, gsp->variable);
+
+		setSlotValueByAttnum(result, newelem, attrno);
```
Comment (gtyun) [inline src/backend/executor/nodeModifyGraph.c]: 'setSlotValueByAttnum' function name is duplicated. It may cause conflict.

**Comment by htlim** [comment]
Thanks for reviewing gtyun.
I modified and committed again.

**Pull request closed without merging**